codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -57,6 +57,46 @@ data Picture
            !Double
```

Comment:
You shouldn't need the `Path` costructor any longer.  `Path` is just an old name for `Polyline` (just like `loop` is an old name for `closedCurve`)

Reply:
But what about the path functions? What should I use in place of those (and functions such as ```path```, ```thickPath```, ```loop``` and ```thickLoop```)? Should I use ```closedCurve``` in their place?

Reply:
`path` is the same as `polyline`
`thickPath` is `thickPolyline`
`loop` is `closedCurve`
`thickLoop` is `thickClosedCurve`

Reply:
By the way, "path" and "loop" are deprecated.  They only exist because students used to use them a long time ago.  I'll probably drop support for them either this summer or next summer.

Reply:
Oh, I see now.  It looks like `n` is sometimes used instead of `w`.  They mean the same thing.